workflows/datree/versions/0.0.1/docs/datree-policy-check.md

```diff
@@ -0,0 +1,89 @@
+# datree-policy-check
+
+## Summary
+Run a policy check on a cloned repository with Datree
+
+## Inputs/Outputs
+
+### Inputs
+* CLI_ARGUMENTS (optional) - Datree CLI arguments as specified here: https://hub.datree.io/setup/cli-arguments#flags
+* DATREE_TOKEN (required) - the k8s secret name that contains a key named token with the datree secret inside it
+* HELM_ARGUMENTS (optional) - The Helm arguments to be used, if the path is a Helm chart
+* INPUT_PATH (required) - File/s to be tested
+* IS_HELM_CHART (optional) - Is the desired path a Helm chart?
+* IS_KUSTOMIZATION (optional) - Is the desired path a Kustomization?
+* KUSTOMIZE_ARGUMENTS (optional) - The Kustomize arguments to be used, if the path is a Kustomization
+* WORKING_DIRECTORY (optional) - The directory to which the repository is cloned. It can be an explicit path in the container’s file system, or a variable that references another step. The default value is '/codefresh/volume/${{CF_REPO_NAME}}'
```

Comment:
/codefresh/volume/${{CF_REPO_NAME}} has no existence in argo workflow
the default should be an artifact (maybe named repo)